core/subprojects/README.md

```diff
@@ -0,0 +1,11 @@
+# core/subprojects
+
+## What's this? What _is_ this?
+
+- These are subprojects accessible to the `core` meson project.
+
+## icu
+
+- Ideally we would use meson's [wrapdb](https://mesonbuild.com/Wrapdb-projects.html), and in fact the files here are sourced from that project.
+- However, it requires a much newer meson version to work properly: 0.55 at least.
```

Comment:
This statement is not quite correct now (given we are 1.0 on all platforms except for linux packaging agents?)

Reply:
I replaced it with a different statement.

Reply:
When I wrote this, I still thought there was a way to customize the build without modifying the patchfiles. There isn't, but maybe someday there will be. (I filed the upstream ticket.)